config/routes.rb

```diff
@@ -1,9 +1,10 @@
 Rails.application.routes.draw do
   root to: 'main#index'
+  mount_devise_token_auth_for 'User', at: 'auth', skip: [:omniauth_callbacks]
   namespace :api do
     namespace :v1 do
-      mount_devise_token_auth_for 'User', at: 'auth', skip: [:omniauth_callbacks]
       resources :open_library, only: [:index, :show], param: :isbn
+      resources :book
```

Comment:
Set only the routes that you use 
```suggestion
      resources :book, only: %i[index show]
```